File: GithubUserJava/app/src/main/java/com/example/myapplication/adapter/SectionPagerAdapter.java
package com.example.myapplication.adapter;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.viewpager2.adapter.FragmentStateAdapter;

import com.example.myapplication.ui.DetailActivity;
import com.example.myapplication.ui.FollowFragment;

public class SectionPagerAdapter extends FragmentStateAdapter {

    private final String username = DetailActivity.username;

    public SectionPagerAdapter(AppCompatActivity activity) {
        super(activity);
    }

    @Override
    public int getItemCount() {
        return 2;
    }

    @Override
    public Fragment createFragment(int position) {
        FollowFragment fragment = new FollowFragment();
        Bundle args = new Bundle();
        args.putInt(FollowFragment.ARG_POSITION, position + 1);
        args.putString(FollowFragment.ARG_USERNAME, username);
        fragment.setArguments(args);
        return fragment;
    }
}

File: GithubUserKotlin/app/src/main/java/com/example/myapplication/adapter/SectionPagerAdapter.kt
package com.example.myapplication.adapter


import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import androidx.viewpager2.adapter.FragmentStateAdapter
import com.example.myapplication.ui.DetailActivity
import com.example.myapplication.ui.FollowFragment

class SectionPagerAdapter(activity: AppCompatActivity) :
    FragmentStateAdapter(activity) {

    val username = DetailActivity.username

    override fun getItemCount(): Int {
        return 2
    }

    override fun createFragment(position: Int): Fragment {
        val fragment = FollowFragment()
        fragment.arguments = Bundle().apply {
            putInt(FollowFragment.ARG_POSITION, position + 1)
            putString(FollowFragment.ARG_USERNAME, username)
        }
        return fragment
    }

}